src/zenml/cli/cli.py

```diff
@@ -62,7 +63,9 @@ def get_help(self, ctx: Context) -> str:
         """
         formatter = ctx.make_formatter()
         self.format_help(ctx, formatter)
-        return formatter.getvalue().rstrip("\n")
+        # TODO [LOW]: Find solution for support console.pager and color support in print
+        rich.print(formatter.getvalue().rstrip("\n"))
```

Comment:
instead of calling `rich.print`, usually we do `from zenml.console import console` and then call `console.print...`. I think you should probably stick to that convention otherwise we have too many different rich consoles out there...